source/exercises/string_formatting.rst

```diff
@@ -130,15 +136,63 @@ Task Three
 
     use string formating to print:
 
-    ``'02 27 2017 04 30``
+    ``'02 27 2017 04 30'``
 
 Hint: use index numbers to specify positions.
 
 
-Task four
+Task Five
 ---------
+f-strings are new to Python (version 3.6), but are very powerful and efficient. This means they are worth understanding and using. And this is made easier than it might be because they use the same, familiar formatting language that is conventionally used in Python (in .format()).
+
+So in this exercise we are going to specifically use f-strings.
+
+Here's the simplest example, just to show the basic syntax:
+
+.. code-block:: ipython
+
+    In [1]: f'a simple f-string'
+    Out[1]: 'a simple f-string'
+
+And here's an example that shows how you can use available variables in a f-string:
+
+.. code-block:: ipython
+
+    In [2]: name = 'Andy'
+    In [3]: f'Your name is {name}'
+    Out[3]: 'Your name is Andy'
+
```

Comment:
Now let's add an example of evaluating an expression in an f-string:

In addition to referencing variables in the local scope, f-strings can evaluate simple expressions in line like so:

```
In [5]: f"Your name is {name.upper()}"
Out[5]: 'Your name is ANDY'

In [6]: name = "andy"

In [7]: f"Your name is {name.upper()}"
Out[7]: 'Your name is ANDY'
```

or 

```
In [8]: a = 5

In [9]: b = 10

In [10]: f"The sum is: {a+b}"
Out[10]: 'The sum is: 15'
```